src/instructlab/model/evaluate.py

```diff
@@ -659,9 +659,17 @@ def evaluate(
                 for i, evaluator in enumerate(evaluators):
                     branch = branches[i]
                     print(f"Evaluating answers for branch {branch}...")
-                    qa_pairs, error_rate = evaluator.judge_answers(
+                    judgement = evaluator.judge_answers(
                         api_base, max_workers=max_workers, serving_gpus=effective_gpus
                     )
+
```

Comment:
nit: maybe a comment here explaining the difference?

Reply:
I'd rather just leave it as is. This commit is just a place holder for the time between when https://github.com/instructlab/eval/pull/138 is merged and https://github.com/instructlab/instructlab/pull/2316 is merged (that PR will remove the logic in this commit).